spring-boot-dixit/src/main/java/tw/wally/dixit/clients/LobbyServiceDriver.java

```diff
@@ -0,0 +1,11 @@
+package tw.wally.dixit.clients;
+
+import tw.wally.dixit.Registration;
+
+/**
+ * @author - wally55077@gmail.com
+ */
+@FunctionalInterface
```

Comment:
Why would this be @FuncionalInterface?
I think there'll be more than one method in the future, e.g., notify the lobby service for a game's over)